extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -234,6 +234,14 @@ export class GoogleSubscriptionsPlatform {
     //TODO: implement this
     return Promise.resolve(false);
   }
+
+  /** @override */
+  decorateUI(element, action, options) {
+    if (action === 'login') {
+      element.innerHTML = '';
+      this.runtime_.buttonApi_.attach(element, options);
```

Comment:
Use `this.runtime_.attachButton(...)` API. One nuance: you must pass a callback. It should be empty in this case. I'll update SwG later to not require it. But for now it's required.

Reply:
Done